pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -30,40 +31,55 @@ def set_model(self, model):
             model = model.module
         self.model = model
 
-    def on_epoch_begin(self, epoch, logs=None):
+    def set_trainer(self, trainer):
+        self.trainer = trainer
+
+    def on_epoch_begin(self):
         """
         called when the epoch begins
-
-        Args:
-            epoch (int): current epoch
-            logs (dict): key-value pairs of quantities to monitor
-
-            Example:
-
-                on_epoch_begin(epoch=2, logs={'val_loss': 0.2})
         """
         pass
 
-    def on_epoch_end(self, epoch, logs=None):
+    def on_epoch_end(self):
+        """
+        called when the epoch ends
+        """
         pass
 
-    def on_batch_begin(self, batch, logs=None):
+    def on_batch_begin(self):
         """
         called when the batch starts.
+        """
+        pass
 
-        Args:
-            batch (Tensor): current batch tensor
-            logs (dict): key-value pairs of quantities to monitor
+    def on_batch_end(self):
+        """
+        called when the batch ends.
```

Comment:
when you use . at the end, make it as a sentence and it can be even a single line, so
```
"""Callback - when the batch ends."""

Reply:
Done.